app/models/competition.rb

```diff
@@ -321,6 +322,18 @@ def has_administrative_notes?
     end
   end
 
+  validate :auto_close_threshold_validations, if: -> { auto_close_threshold.present? }
```

Comment:
`if: :auto_close_threshold?` should do:
![image](https://github.com/user-attachments/assets/8189756e-11cd-4ecd-acc4-e53a58067562)